contents/docs/product-analytics/web-vitals.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Web vitals autocapture
+sidebar: Docs
+showTitle: true
+availability:
+  free: full
+  selfServe: full
+  enterprise: full
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import { ProductVideo } from 'components/ProductVideo'
+
+PostHog can automatically capture [core web vitals](https://web.dev/explore/learn-core-web-vitals) like LCP, FID, CLS, and FCP. This you means you don't need to manually add logging from [Google's web vitals library](https://github.com/GoogleChrome/web-vitals).
```

Comment:
- We should spell out the acronyms the first time we use them
- Linking out to Google's library might only confuse people, don't think it is necessary. 
```suggestion
PostHog can automatically capture [core web vitals](https://web.dev/explore/learn-core-web-vitals) like largest contentful paint, first input delay, cumulative layout shift, and first contentful paint. This you means you don't need to manually add web vitals logging.
```